docs/content/guides/developer/app-examples/escrow.mdx

```diff
@@ -1,6 +1,31 @@
 ---
 title: Escrow
-draft: true
+hide_table_of_contents: true
 ---
 
-Content coming soon
\ No newline at end of file
+This guide demonstrates an atomic swap, which does not require a trusted third party, by utilizing a [shared object](/concepts/object-ownership/shared) as an escrow. Shared objects are a unique concept to Sui. They are able to be modified by any transaction and by any signer, given they meet the requirements set forth by the package that defined the type.
+
+:::info
+
+See [Shared versus Owned Objects](/guides/developer/sui-101/shared-owned) for more information on the differences between object types.
+
+:::
+
+The guide is split into three parts:
+
+1. [Backend](/guides/developer/app-examples/escrow-backend): The smart contracts that hold the state and perform the swaps.
+1. [Indexing and API Service](/guides/developer/app-examples/escrow-indexer-api): A service that indexes chain state to discover trades, and an API service to read this data.
+1. [Frontend](/guides/developer/app-examples/escrow-frontend): Enables users to list objects for sale and to accept trades.
```

Comment:
Nit: on nomenclature here -- to me it reads a little strange to have "smart contracts" (also are we supposed to use the term "smart contracts"?) in the "Backend" section, vs Backend meaning "Indexing and API service". Maybe these are the standard names we use in other guides, if so, feel free to ignore me, but I would name this something like:

- On-chain
- Backend
- Frontend

Reply:
@amnn I feel like we should make sure to differentiate the indexer here, as it's write-only service. Even though I've kept it under the same folder to re-use the schema + db connection logic, it could easily be used for write-only purposes!